packages/core/src/routines/CreateConfigRoutine.ts

```diff
@@ -118,6 +133,62 @@ export default class CreateConfigRoutine<Ctx extends ConfigContext> extends Rout
     return configPath;
   }
 
+  /**
+   * Create a configuration file for the defined driver based on a custom template
+   * provided by the consumer.
+   */
+  @Bind()
+  async createConfigFileFromTemplate(context: Ctx, configs: ConfigObject[]): Promise<Path> {
+    const { driver, tool } = this.options;
+    const { metadata } = driver;
+    const configPath = context.cwd.append(metadata.configName);
```

Comment:
Not sure if this makes sense, but might be nice to be able to configure this, I know a lot of tools support multiple formats/locations for config files.  Also possible drivers already support overwriting metadata, in which case maybe this doesn't matter.  I could imagine templates being useful for generating config files that might not be supported out of the box (eg `.eslintrc.yml`) or generating your configs to a location other than the project root (eg. writing the config files to `./config/.eslintrc.js`

Reply:
@hayes  How would the actual driver find the config then when it's being ran if the config is missing (because it's been named something different)?

Reply:
Those can also be done with events, once I add the event code.

Reply:
for things `.eslintrc.yml` a lot of tool use https://github.com/davidtheclark/cosmiconfig or something similar.  I don't think the driver actually needs to know the location, for custom paths, I would imagine you would pair it with a `--config` argument when you set up the driver.  I don't feel strongly about this, just seemed interesting/useful